File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Dps.java
package com.BlizzardArmory.model.warcraft.equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Dps.
 */
public class Dps {

    @SerializedName("value")
    @Expose
    private float value;
    @SerializedName("display_string")
    @Expose
    private String displayString;

    /**
     * Gets value.
     *
     * @return the value
     */
    public float getValue() {
        return value;
    }

    /**
     * Sets value.
     *
     * @param value the value
     */
    public void setValue(float value) {
        this.value = value;
    }

    /**
     * Gets display string.
     *
     * @return the display string
     */
    public String getDisplayString() {
        return displayString;
    }

    /**
     * Sets display string.
     *
     * @param displayString the display string
     */
    public void setDisplayString(String displayString) {
        this.displayString = displayString;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Dps.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Dps.
 */
@Keep
data class Dps(

    @SerializedName("value")
    var value: Float,

    @SerializedName("display_string")
    var displayString: String

)